yaroslav.nazarenko/first/task1/task1.rb

```diff
@@ -0,0 +1,5 @@
+x = Array.new(3) { Array.new(3) { rand (-10..10) } }
+puts "Random array: #{x}"
+x.flatten.each.select { |x| puts x if x > 0 }
+pos_num = x.flatten(&:positive?).sum
```

Comment:
```suggestion
pos_num = x.flatten.each(&:positive?).sum
```
it seems like you deleted a method which you shouldn't have been deleted